src/opcuabridge/image.h

```diff
@@ -0,0 +1,57 @@
+#ifndef OPCUABRIDGE_IMAGE_H_
+#define OPCUABRIDGE_IMAGE_H_
+
+#include "hash.h"
+
+#include "common.h"
+
+namespace opcuabridge {
+class Image {
+ public:
+  Image() {}
+  virtual ~Image() {}
+ public:
+  const std::string& get_filename() const { return filename_; }
+  void set_filename(const std::string& filename) { filename_ = filename; }
+  const std::size_t& get_length() const { return length_; }
+  void set_length(const std::size_t& length) { length_ = length; }
+  const std::size_t& get_numberOfHashes() const { return numberOfHashes_; }
+  void set_numberOfHashes(const std::size_t& numberOfHashes) { numberOfHashes_ = numberOfHashes; }
+  const std::vector<Hash>& get_hashes() const { return hashes_; }
```

Comment:
Can numberOfHashes every be different to the length of get_hashes()

Reply:
Interface conforms to uptane spec. And it may be still useful when hashes implementation get changed.

Reply:
Yeah, I've seen that 'numberOfHashes' field in the Uptane ASN.1 spec.  My belief is that it comes from a misunderstanding of how ASN.1 works, since ASN.1 includes length information anyway. My question though is what would it mean to have numberOfHashes != hashes_.size().  If this isn't allowed, then why not just call hashes_.size() when required?

Reply:
I'm sure that numberOfHashes == hashes_.size() is always true. So, now we can conclude with removing get_/set_numberOfHashes due to the fact that std::vector container implementation is selected. Once the implementation changed to array, raw pointer or something else we could easily bring numberOfHashes back.